src/WGet.NET/HelperClasses/ProcessOutputReader.cs

```diff
@@ -70,13 +71,18 @@ public static List<WinGetPackage> ToPackageList(string[] output)
             {
                 // [var1..var2] : selects the index range from var1 to var2
                 // (eg. if var1 is 2 and var2 is 5, the selectet index range will be [2, 3, 4])
-                resultList.Add(
+                if (output[i].Contains("upgrades available") || output[i].Contains("version numbers that")) { continue; }
```

Comment:
This should be changed to check if "output[i]" is >= extraInfoStartIndex

Reply:
About the second string im checking for, here is the full line.
`2 package(s) have version numbers that cannot be determined. Use "--include-unknown" to see all results.`
This doesnt always occur. As it states, only for packages whos version cannot be determined.
I hope this helps

Reply:
Ill have a look at this later, thanks for the hint tho!

Reply:
Im not entirely sure what you meant to compare here, perhaps the length of each line?
In that case itd be `if (output[i].Length >= extraInfoStartIndex)` which doesnt seem to work.
Maybe i just didnt understand what you meant. Feel free to let me know if thats the case

Reply:
Oh that was my error sorry. I was thinking about putting the action in the if brackets but didn't mention it. If we do it with the `continue` in there to skip, it should be like this: `output[i].Length > extraInfoStartIndex`.
With that, the entry should be skipt if the length value is less than the value of the last index we try to access.

Reply:
In my case this didnt work. Ill have a look if theres another way to determine when the line is invalid